Group1/src/main/java/com/stuypulse/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,83 @@
+package com.stuypulse.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.stuypulse.robot.constants.Ports;
+import com.stuypulse.robot.constants.Settings;
+import com.stuypulse.stuylib.control.Controller;
+import com.stuypulse.stuylib.control.feedback.PIDController;
+import com.stuypulse.stuylib.control.feedforward.Feedforward;
+import com.stuypulse.stuylib.math.Angle;
+import com.stuypulse.stuylib.network.SmartAngle;
+
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import static com.stuypulse.robot.constants.Settings.Intake.PID.*;
+import static com.stuypulse.robot.constants.Settings.Intake.FF.*;
+
+/**
+ * Intake Fields:
+ * MotorControllerGroup driver
+ * Motor deploy
+ * Controller controller (PID and FeedForward)
+ * targetAngle
+ * deployEncoder
+ * 
+ * Intake Methods:
+ * setAngle()
+ * retract()
+ * extend()
+ * acquire()
+ * deacquire()
+ */
+public class Intake extends SubsystemBase {
+    private final MotorControllerGroup driver;
+    private final CANSparkMax deploy;
+    private final Encoder encoder;
```

Comment:
Figure out what encoder we are using

Reply:
can spark max is chill but i feel like if ur group can put an absolute encoder in there it would be evenmore based cause then it doesnt matter where ur intake is turned on from yk? 

for the intake the absolute encoder isnt CRITICAL because it must start inside the frame perimeter anyway so you kinda know where it's starting